File: sample/src/main/java/com/benny/app/sample/network/service/douban/model/Movie.java
package com.benny.app.sample.network.service.douban.model;

import com.google.gson.annotations.SerializedName;

/**
 * Created by benny on 1/28/16.
 */
public class Movie {
    public String id;
    public String title;

    public Rating rating;

    @SerializedName("original_title")
    public String originalTitle;

    public String subtype;
    public String summary;

    @SerializedName("reviews_count")
    public int reviewsCount;

    @SerializedName("wish_count")
    public int wishCount;

    @SerializedName("comments_count")
    public int commentsCount;

    @SerializedName("ratings_count")
    public int ratingsCount;

    @SerializedName("douban_site")
    public String doubanSite;

    public String year;
    public Avatars images;
    public String alt;

    @SerializedName("mobile_url")
    public String mobileUrl;

    @SerializedName("share_url")
    public String shareUrl;

    @SerializedName("schedule_url")
    public String scheduleUrl;

    public String[] genres;

    public String[] countries;

    public Cast[] casts;

    public Cast[] directors;

    public String[] aka;
}

File: sample/src/main/kotlin/com/benny/app/sample/network/service/douban/model/Movie.kt
package com.benny.app.sample.network.service.douban.model

import com.google.gson.annotations.SerializedName

/**
 * Created by benny on 1/28/16.
 */
data class Movie (
    var id: String,
    var title: String,

    var rating: Rating,

    @SerializedName("original_title")
    var originalTitle: String,

    var subtype: String,
    var summary: String,

    @SerializedName("reviews_count")
    var reviewsCount: Int = 0,

    @SerializedName("wish_count")
    var wishCount: Int = 0,

    @SerializedName("comments_count")
    var commentsCount: Int = 0,

    @SerializedName("ratings_count")
    var ratingsCount: Int = 0,

    @SerializedName("douban_site")
    var doubanSite: String,

    var year: String,
    var images: Avatars,
    var alt: String,

    @SerializedName("mobile_url")
    var mobileUrl: String,

    @SerializedName("share_url")
    var shareUrl: String,

    @SerializedName("schedule_url")
    var scheduleUrl: String,

    var genres: Array<String>,

    var countries: Array<String>,

    var casts: Array<Cast>,

    var directors: Array<Cast>,

    var aka: Array<String>
)